example.js

```diff
@@ -1,44 +1,58 @@
 const Ably = require("ably");
 
-const ApiKey = "INSERT-YOUR-API-KEY-HERE", /* Add your API key here */
-  clientId = "my-client-id"; /* This is who you will appear as in the presence set */
-if (ApiKey.indexOf("INSERT") === 0) { throw("Cannot run without an API key. Add your key to example.js"); }
+const apiKey = "YOUR_ABLY_API_KEY"; // Replace with your API key
+const randomId =
+  Math.random().toString(36).substring(2, 10) +
+  Math.random().toString(36).substring(2, 10);
```

Comment:
A simplified alternative.
```suggestion
const randomId = Math.random().toString(36).slice(-10);
```

Reply:
Nice